aws_lambda/policy_checks.py

```diff
@@ -446,6 +472,8 @@ def get_repository_data(gh: github_interface, org: str) -> list[dict] | str:
                     repos = repos_response.json()
 
                     for repo in repos:
+                        codeowners_missing = check_file_exists(repo["contents_url"], gh, [".github/CODEOWNERS", "CODEOWNERS", "docs/CODEOWNERS"])
```

Comment:
It is a bit weird that file_exists returns True when it doesn't exist?

Reply:
renamed function to check_file_missing